docker/entrypoint.sh

```
@@ -33,15 +33,22 @@ else
+
+# when running behind a proxy
+if [[ ${BEHIND_PROXY} == "true" ]]; then
+  echo "running behind proxy"
+  echo "listening on port 80 internally"
+  export PARSED_LE_EMAIL=""
+  export PARSED_APP_URL=":80"
+  export APP_ASSET=${APP_URL}
+fi
 
 ## disable caddy if SKIP_CADDY is set
 if [[ "${SKIP_CADDY:-}" == "true" ]]; then
```

Shellcheck (0.11.0)

[error] In dash, [[ ]] is not supported.

(SC3010)